services/minespace-web/src/components/dashboard/mine/projects/ProjectStagesTable.js

```diff
@@ -32,28 +32,51 @@ export class ProjectStagesTable extends Component {
     {
       title: "",
       dataIndex: "stage_status",
-      render: (text, record) => (
-        <div title="Stage Status">
-          <b>{`[${record.stage_status_hash[text]}]` || "N/A"}</b>
-        </div>
-      ),
+      render: (text, record) => {
+        const status =
+          record.stage_status === undefined ? "Not Started" : record.stage_status_hash[text];
+        return (
+          <div title="Stage Status">
+            <b>{status || "N/A"}</b>
+          </div>
+        );
+      },
     },
     {
       title: "",
       dataIndex: "stage",
       align: "right",
-      render: (text, record) => (
-        <Link
-          to={routes.EDIT_PROJECT_SUMMARY.dynamicRoute(
-            record.stage?.payload?.project_guid,
-            record.stage?.payload?.project_summary_guid
-          )}
-        >
-          <Button className="full-mobile margin-small" type="secondary">
-            Edit
-          </Button>
-        </Link>
-      ),
+      render: (text, record) => {
```

Comment:
Could we add the links for each stage to the projectStages payload(begins line 77) in the ProjectOverviewTab component? Then we can just return the record link in this render function and be done with it.

I think this will clean things up, especially when we begin to add further stages..